skins/oasis/modules/PageHeaderController.class.php

```diff
@@ -64,7 +61,7 @@ protected function prepareActionButton() {
 		}
 
 		// If cascade protected, show viewsource button - BugId:VE-89
-		if ( isset( $this->content_actions['edit'] ) && $wgTitle->isCascadeProtected() ) {
+		if ( isset( $this->content_actions['edit'] ) && $this->wg->Title->isCascadeProtected() ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)